SimPEG/potential_fields/base.py

```diff
@@ -70,6 +78,76 @@ def __init__(self, mesh, **kwargs):
             zn1, zn2 = bsw[:, 2], tne[:, 2]
             self.Zn = projection.T * np.c_[mkvc(zn1), mkvc(zn2)]
 
+    @property
+    def store_sensitivities(self):
+        """Options for storing sensitivities.
+
+        There are 3 options:
+
+        - 'ram': sensitivity matrix stored in RAM
+        - 'disk': sensitivities written and stored to disk
+        - 'forward_only': sensitivities are not store (only use for forward simulation)
+
+        Returns
+        -------
+        str
+            A string defining the model type for the simulation.
+            One of {'disk', 'ram', 'forward_only'}.
+        """
+        if self._store_sensitivities is None:
+            self._store_sensitivities = 'ram'
+        return self._store_sensitivities
+
+    @store_sensitivities.setter
+    def store_sensitivities(self, value):
+        choices = ["disk", "ram", "forward_only"]
+        value = value.lower()
+        if value not in choices:
+            raise ValueError(
+                "Store sensitivities option ({}) unrecognized. ",
+                "Choose one of ['disk', 'ram', 'forward_only']".format(value)
+            )
+        self._store_sensitivities = value
+
+    @property
+    def ind_active(self):
+        if self._ind_active is None:
+            self._ind_active = np.asarray(range(self.mesh.nC))
+        return self._ind_active
+
+    @ind_active.setter
+    def ind_active(self, input_array):
+        if not isinstance(input_array, (tuple, list, np.ndarray)):
+            raise TypeError(
+                "'ind_active' must be set using a tuple, list or numpy.ndarray"
+            )
+        if isinstance(input_array, (tuple, list)):
+            input_array = np.array(input_array)
+        if not isinstance(input_array.dtype, (int, bool)):
+            raise TypeError("'ind_active' must be an array of int or bool")
+
+        self._ind_active = input_array
+
+    @property
+    def actInd(self):
+        warnings.warn(
+            "The 'actInd' property has been deprecated. "
+            "Please use 'ind_active'. This will be removed in version 0.17.0 of SimPEG.",
+            FutureWarning,
+        )
+        return self._ind_active
```

Comment:
@jcapriot Do you think we could handle all of these properties -> @property with something like `code_utils.deprecate_property` , but would actually re-route to the setter of the new name? Feels like it would simplify the classes during transition.